server/journeys/referral/routes.ts

```diff
@@ -4,26 +4,29 @@ import { ReferralRootController } from './controller'
 import { OnBehalfOfRoutes } from './on-behalf-of/routes'
 import CsipApiService from '../../services/csipApi/csipApiService'
 import { ReferralAreaOfWorkRoutes } from './area-of-work/routes'
+import PrisonerSearchService from '../../services/prisonerSearch/prisonerSearchService'
+import StartJourneyRoutes from './start/routes'
 
-function Routes(): Router {
-  const router = Router()
+function Routes(csipApiService: CsipApiService): Router {
+  const router = Router({ mergeParams: true })
 
   const get = (path: string, handler: RequestHandler) => router.get(path, asyncMiddleware(handler))
 
   const referralRootController = new ReferralRootController()
 
   get('/', referralRootController.GET)
 
+  router.use('/on-behalf-of', OnBehalfOfRoutes())
+  router.use('/area-of-work', ReferralAreaOfWorkRoutes(csipApiService))
+
   return router
 }
 
-export default function routes(csipApiService: CsipApiService): Router {
+export default function routes(csipApiService: CsipApiService, prisonerSearchService: PrisonerSearchService): Router {
   const router = Router({ mergeParams: true })
 
-  router.use('/', Routes())
-
-  router.use('/on-behalf-of', OnBehalfOfRoutes())
-  router.use('/area-of-work', ReferralAreaOfWorkRoutes(csipApiService))
+  router.use('/', StartJourneyRoutes(prisonerSearchService))
```

Comment:
i was thinking `/start/routes.ts` can be a base class to be inherited, so here we can make a call like:
```
router.get('/prisoners/:prisonerNumber/referral/start', this.redirectWithPrisonerData('/referral/on-behalf-of'))
```

then we just have one unit test for the base class instead of repeating it for every journey.

but then we only have one journey at the moment, so it doesn't matter.

Reply:
I think it would end up being more confusing and less obvious what's going on having hidden routes in parent classes, but perhaps let's see where we end up after some more journeys?

We would need a unit test for each journey anyway to ensure that the start route redirects to the specific journey route, imo